glib/src/gstring.rs

```diff
@@ -445,6 +445,12 @@ impl fmt::Debug for GString {
     }
 }
 
+impl Default for GString {
+    fn default() -> Self {
+        Self(Inner::Native(Some(CString::default())))
```

Comment:
You probably want to create a "foreign" empty string here. The way you'd potentially have fewer copies when actually using it with C APIs and it's in line with the other code here.